tests/runner/replicate/output_1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "scilla_major_version": "0",
+  "gas_remaining": "7946",
+  "_accepted": "false",
+  "messages": [],
+  "states": [ { "vname": "_balance", "type": "Uint128", "value": "0" } ],
+  "events": [
+    {
+      "_eventname": "Replicated",
+      "params": [
+        {
+          "vname": "new_addr",
+          "type": "ByStr20",
+          "value": "0x11345678901234567890123456789012345678aa"
+        }
+      ]
+    }
+  ]
```

Comment:
It bothers me that the output file doesn't contain information about the newly deployed contract. I'm guessing you've put that information in blockchain_1.json, but we should have a way to track the state changes that happen to the blockchain outside of the current contract. I'm not sure how to do it, though.

Reply:
> REPLICATE_CONTRACT does two things: It deploys a contract, and it returns an address. We currently only test the latter, but we ought to be able to test the former as well.
> 
> In some sense what ought to happen is that the `_external` part of `state.json` ought to change, but it's not clear to me how we should output that change.

It can be tested from the blockchain unit test suite quite easily. Doing it on our side might be more complex.

Reply:
I'm ok with it being tested on the C++ side, as long as we actually do it.